### PR TITLE
remove: shuffle=False option

### LINKED ISSUE #6885: DistributedWeightedRandomSampler remove `shuffle=False` option
**Describe the bug**
https://github.com/Project-MONAI/MONAI/blob/4cd717ca830ed7cd811144d96675fdbe586633a4/monai/data/samplers.py#L98
the random sampler will always generate 'shuffled' outputs, the `shuffle=False` option is not valid and should be removed

### PR DESCRIPTION
**Pull request by AmirAflak**
Fixes #6885 

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/data/samplers.py [hunk 1]
```diff
@@ -95,15 +95,14 @@ def __init__(
         even_divisible: bool = True,
         num_replicas: int | None = None,
         rank: int | None = None,
-        shuffle: bool = True,
         **kwargs,
     ):
+        kwargs.setdefault("shuffle", True)
```
Comment (wyli) [inline monai/data/samplers.py:100]: thanks! please update the docstring and check the test case https://github.com/Project-MONAI/MONAI/blob/4cd717ca830ed7cd811144d96675fdbe586633a4/tests/test_weighted_random_sampler_dist.py#L26

Comment (AmirAflak) [reply monai/data/samplers.py:100]: Checked the test case:
image

**Review by wyli — APPROVED** [review]

**Pull request merged**